src/templates/media_querry.css

```diff
@@ -0,0 +1,242 @@
+:root {
```

Comment:
Kindly move this file into the static folder

Reply:
alright.